wasm/gcc-loops/gcc-loops.cpp

```diff
@@ -369,18 +369,20 @@ int main(int argc,char* argv[]){
   BENCH("Example1",   example1(), Mi*10, digest_memory(&a[0], &a[256]));
   BENCH("Example2a",  example2a(N, 2), Mi*4, digest_memory(&b[0], &b[N]));
   BENCH("Example2b",  example2b(N, 2), Mi*2, digest_memory(&a[0], &a[N]));
-  BENCH("Example3",   example3(N, ia, ib), Mi*2, digest_memory(&ia[0], &ia[N]));
-  BENCH("Example4a",  example4a(N, ia, ib), Mi*2, digest_memory(&ia[0], &ia[N]));
-  BENCH("Example4b",  example4b(N-10, ia, ib), Mi*2, digest_memory(&ia[0], &ia[N]));
-  BENCH("Example4c",  example4c(N, ia, ib), Mi*2, digest_memory(&ib[0], &ib[N]));
+  // dlehmann: Disable the following four examples, since Emscripten warns:
+  // gcc-loops.cpp:373:35: warning: passing 4-byte aligned argument to 16-byte aligned parameter 2 of 'example3' may result in an unaligned pointer access [-Walign-mismatch]
+  // BENCH("Example3",   example3(N, ia, ib), Mi*2, digest_memory(&ia[0], &ia[N]));
+  // BENCH("Example4a",  example4a(N, ia, ib), Mi*2, digest_memory(&ia[0], &ia[N]));
+  // BENCH("Example4b",  example4b(N-10, ia, ib), Mi*2, digest_memory(&ia[0], &ia[N]));
+  // BENCH("Example4c",  example4c(N, ia, ib), Mi*2, digest_memory(&ib[0], &ib[N]));
   BENCH("Example7",   example7(4), Mi*4, digest_memory(&a[0], &a[N]));
   BENCH("Example8",   example8(8), Mi/4, digest_memory(&G[0][0], &G[0][N]));
   BENCH("Example9",   example9(&dummy), Mi*2, dummy);
   BENCH("Example10a", example10a(sa,sb,sc,ia,ib,ic), Mi*2, digest_memory(&ia[0], &ia[N]) + digest_memory(&sa[0], &sa[N]));
   BENCH("Example10b", example10b(sa,sb,sc,ia,ib,ic), Mi*4, digest_memory(&ia[0], &ia[N]));
   BENCH("Example11",  example11(), Mi*2, digest_memory(&d[0], &d[N]));
   BENCH("Example12",  example12(), Mi*4, digest_memory(&a[0], &a[N]));
-  //BENCH("Example21",  example21(ia, N), Mi*4, digest_memory(&ia[0], &ia[N]));
+  BENCH("Example21",  example21(ia, N), Mi*4, digest_memory(&ia[0], &ia[N]));
```

Comment:
Note that those sub-items were a bit of a mess before: Even though `Example21` was the only one that was commented out in the source file, I looked at the old binary and realized only a small subset was actually run, namely `Example1`, `Example3`, `Example4a`, `Example7`, `Example10a`, and `Example11`.
@kmiller68 Do you know why this subset was chose in the past?
I think the source file should reflect what we actually run and measure, so I opted for enabling all sub-items (except the four with warnings, see above). Does that make sense to you?

Reply:
Hmm, I wonder if that was accidental 😬? That seems fine to me though. Although, as stated above I'm happy to leave the above cases since they should be fine within wasm.